### ISSUE TITLE
[Feature Request] Supporting YouTube playlists

### ISSUE THREAD

**Issue by Harmonia2Side** [g-fb/haruna#41]
Is is possible to support playing playlists directly from a youtube playlist link?
When I select 'openURL' and paste a youtube link to a playlist instead of a video, it plays the first video on the playlist, then crashes, so I'm assuming this feature is not implemented as of yet

**Comment by g-fb** [comment]
Added.

You can try it by installing the beta, or compiling yourself.
```
flatpak remote-add flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install flathub-beta com.georgefb.haruna
flatpak run --branch=beta com.georgefb.haruna
```

**Comment by Harmonia2Side** [comment]
OMG It has actually been done, thank you!! I'll give it a try right away (^^)

**Comment by Harmonia2Side** [comment]
Sorry to disturb again and sorry it took so long! Just passing to say that the feature works perfectly, the sidebar to select track on the playlist is very gorgeous/useful, it's extremely amazing that this actually was done and that I'll probably be using this as my main youtube interface from now on. Thank you guys once again, cheers and best regards (^^)

**Comment by logix2** [comment]
Hi. This feature works as long as the playlist URL doesn't include a video. If a video is included in the playlist URL, e.g. https://www.youtube.com/watch?v=PsO6ZnUZI0g&list=PLdYwhvDpx0FI2cmiSVn5cMufHjYHpo_88 , Haruna doesn't show the playlist in the overlay, and choosing "Play Next" crashes Haruna.

**Comment by g-fb** [comment]
Only playlist links are supported `https://www.youtube.com/playlist?list=id`.

> Haruna doesn't show the playlist

Because the playlist is empty, so there's nothing to show.

> choosing "Play Next" crashes Haruna

This should be fixed 02da59a6c6760f375e8700e50631a64f0e776a7c
